catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/web/metrics/IcebergMetricsStorage.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright 2023 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics;
+
+import java.io.IOException;
+import java.time.Instant;
+import org.apache.iceberg.metrics.MetricsReport;
+
+/** A storage API to store Iceberg metrics. */
+public interface IcebergMetricsStorage {
+
+  /**
+   * The string that represents the type of IcebergMetricsStorage.
+   *
+   * @return The string that represents the type of IcebergMetricsStorage.
+   */
+  String shortName();
```

Comment:
This seems only used for log display, seems not so useful.

Reply:
removed